aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -33,10 +33,13 @@ module aptos_framework::voting {
     use aptos_framework::timestamp;
     use aptos_framework::transaction_context;
 
-    /// Error codes.
+    /// Current script's execution has does not match the specified proposal's.
```

Comment:
`execution hash` missing the h

`Current script's execution hash does not match the proposal's hash`